### PR TITLE
Plat 8475 docker

### PR DESCRIPTION
**Pull request by kalturaguy**

### CODE REVIEW

**Review by david-winder-kaltura — APPROVED** [review]

File: KalturaWowzaServer/src/main/java/com/kaltura/media_server/services/DiagnosticsProvider.java [hunk 1]
```diff
@@ -78,6 +88,29 @@ public void execute(HashMap<String,Object> data, HashMap<String,String > quarySt
             data.put("dateStarted", dateStarted);
             data.put("timeRunning", timeRunning);
             data.put("hostName", hostName);
+            data.put("CpuUsage", CpuUsage);
```
Comment (kalturaguy) [inline KalturaWowzaServer/src/main/java/com/kaltura/media_server/services/DiagnosticsProvider.java]: 1.should be camel case (small C) - at the end it's serialize to Json
2. the value should be a double value and not a string (otherwise hard to parse)

File: KalturaWowzaServer/src/main/java/com/kaltura/media_server/services/DiagnosticsProvider.java [hunk 2]
```diff
@@ -78,6 +88,29 @@ public void execute(HashMap<String,Object> data, HashMap<String,String > quarySt
...
+            int in = 0, out = 0;
+            for (IMediaStream stream : appInstance.getStreams().getStreams()) {
+                if (!( stream instanceof MediaStreamLive))
+                    continue;
+                if (stream.isTranscodeResult())
+                    out++;
+                else
+                    in++;
+            }
+            entriesData.put("Input Streams", in);
```
Comment (kalturaguy) [inline KalturaWowzaServer/src/main/java/com/kaltura/media_server/services/DiagnosticsProvider.java]: we just want the number of inputs and outputs

File: KalturaWowzaServer/src/main/java/com/kaltura/media_server/services/DiagnosticsProvider.java [hunk 3]
```diff
@@ -78,6 +88,29 @@ public void execute(HashMap<String,Object> data, HashMap<String,String > quarySt
...
+            entriesData.put("OutPut Streams", out);
+
+            Set<String> playingEntriesList = Utils.getEntriesFromApplication(appInstance);
+            entriesData.put("Entries Count", playingEntriesList.size());
```
Comment (kalturaguy) [inline KalturaWowzaServer/src/main/java/com/kaltura/media_server/services/DiagnosticsProvider.java]: camel case..

File: KalturaWowzaServer/src/main/java/com/kaltura/media_server/services/DiagnosticsProvider.java [hunk 4]
```diff
@@ -78,6 +88,29 @@ public void execute(HashMap<String,Object> data, HashMap<String,String > quarySt
...
+                    continue;
+                if (stream.isTranscodeResult())
+                    out++;
+                else
+                    in++;
+            }
+            entriesData.put("Input Streams", in);
+            entriesData.put("OutPut Streams", out);
+
+            Set<String> playingEntriesList = Utils.getEntriesFromApplication(appInstance);
```
Comment (kalturaguy) [inline KalturaWowzaServer/src/main/java/com/kaltura/media_server/services/DiagnosticsProvider.java]: you already do a pass over all streams above, it's double the work... (+ calling appInstance.getStreams() twice)
i think it's better to just copy the line that count the entries
